dcrwallet.go

```diff
@@ -142,86 +196,78 @@ func walletMain() error {
 		//
 		// Until then, since --noinitialload users are expecting to use
 		// the wallet only over RPC, disable this feature for them.
-		if !cfg.NoInitialLoad {
-			if cfg.Pass != "" {
-				passphrase = []byte(cfg.Pass)
-				w.SetInitiallyUnlocked(true)
-				var unlockAfter <-chan time.Time
-				err = w.Unlock(passphrase, unlockAfter)
-				if err != nil {
-					log.Errorf("Incorrect passphrase in pass config setting.")
-					return err
-				}
-			} else {
-				passphrase = startPromptPass(w)
+		if cfg.Pass != "" {
+			err = w.Unlock([]byte(cfg.Pass), nil)
```

Comment:
You need to add back:

```go
passphrase = []byte(cfg.Pass)
```

otherwise ticket buying with cfg.Pass will be broken.